src/Altinn.Apps/AppTemplates/AspNet/App/logic/Validation/ValidationHandler.cs

```diff
@@ -1,17 +1,23 @@
-using System;
 using System.Threading.Tasks;
 using Altinn.Platform.Storage.Interface.Models;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc.ModelBinding;
-// using Altinn.App.Models; // Uncomment this line to refer to app model(s)
 
+//// using Altinn.App.Models; // Uncomment this line to refer to app model(s)
```

Comment:
I don't see why we can't include this as default.

Reply:
The namespace doesn't exist before a model has been generated.